src/providers/openai_realtime.py

```
@@ -622,6 +643,34 @@ async def send_audio(self, audio_chunk: bytes, sample_rate: int = None, encoding
+            return
+        try:
+            async with self._audio_lock:
+                self._pending_audio_provider_rate.clear()
+            if self.websocket and self.websocket.state.name == "OPEN":
+                await self._send_json(
+                    {
+                        "type": "input_audio_buffer.clear",
+                        "event_id": f"clear-greeting-{uuid.uuid4()}",
+                    }
+                )
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)